MinimedKit/BatteryChemistryType.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  BatteryChemistryType.swift
+//  RileyLink
+//
+//  Created by Jeremy Lucas on 11/15/16
+//  Copyright © 2016 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+enum BatteryChemistryType: Int, CustomStringConvertible {
+    case alkaline = 0
+    case lithium
+
+    var description: String {
+        switch self {
+        case .alkaline:
+            return NSLocalizedString("Alkaline", comment: "Describing the battery chemistry as Alkaline")
+        case .lithium:
+            return NSLocalizedString("Lithium", comment: "Describing the battery chemistry as Lithium")
+        }
+    }
+
+    var maxVoltage: Double {
+        switch self {
+        case .alkaline:
+            return 1.58
+        case .lithium:
+            return 1.58
+        }
+    }
+
+    var minVoltage: Double {
+        switch self {
+        case .alkaline:
+            return 1.26
+        case .lithium:
+            return 1.32
+        }
+    }
+
+    func chargeRemaining(voltage: Double) -> Double {
```

Comment:
Not sure I follow; right now it is explicit: i.e. callers must specify `voltage:` as the label when calling. Are you saying to do `_ voltage:` to make it implicit?